### PR TITLE
Morgabra/add decrypt

### PR DESCRIPTION
**Pull request by jirwin**

### CODE REVIEW

File: pkg/client/appuser.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+		return nil, err
+	}
+
+	if err := NewHTTPError(resp.RawResponse); err != nil {
+		return nil, err
+	}
+
+	// appUsers := make([]shared.AppUser, 0)
+
+	return resp.AppUserServiceListCredentialsResponse.List, nil
```
Comment (mstanbCO) [inline pkg/client/appuser.go:44]: Probably should check that resp.appuserServiceListCredentialsResponse is not nil before using the list.

File: pkg/client/appuser.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	resp, err := c.sdk.AppUser.ListAppUserCredentials(ctx, operations.C1APIAppV1AppUserServiceListAppUserCredentialsRequest{AppID: appID, AppUserID: appUserID})
+	if err != nil {
+		return nil, err
+	}
+
+	if err := NewHTTPError(resp.RawResponse); err != nil {
+		return nil, err
+	}
+
+	// appUsers := make([]shared.AppUser, 0)
```
Comment (mstanbCO) [inline pkg/client/appuser.go:42]: remove the comments in this function?

Comment (phoebesimon) [reply pkg/client/appuser.go]: @mstanbCO removed in [55e1e3f](https://github.com/ConductorOne/cone/pull/108/commits/55e1e3f8fd946143ea4f1c340a12efbdde542824)

File: pkg/client/appuser.go (hunk 1 above)
Comment (phoebesimon) [reply pkg/client/appuser.go:47]: @mstanbCO checking nil values in [55e1e3f](https://github.com/ConductorOne/cone/pull/108/commits/55e1e3f8fd946143ea4f1c340a12efbdde542824)

**Review by mstanbCO — APPROVED** [review]

**Pull request merged**